tools/tasks/gulp-pa11y.js

```diff
@@ -0,0 +1,92 @@
+const fs = require('fs');
+const path = require('path');
+
+const pa11y = require('pa11y');
+const reporter = require('pa11y-reporter-cli'); // As pa11y 5 stabilizes, we can pull this off pa11y
+const _ = require('lodash');
+const through = require('through2');
+
+// Grab project root pa11y.json as object
+const pallyJson = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'pa11y.json')));
+// Custom logging tools for pa11y
+const logConfig = {
+  log: {
+    debug: console.log,
+    error: console.error,
+    info: console.log,
+  },
+};
+// Smash them together
+const pa11yConfig = _.assign(pallyJson, logConfig);
+
+/**
+ * Run pa11y testing one after the next
+ * @returns {*}
+ */
+function pa11yConcurrent() {
```

Comment:
Don't you mean pa11ySequential or in node terminology `pa11ySync` short for pa11ySynchronous?

Reply:
BLARG, I meant "consecutive".